hack/chains/README.md

```diff
@@ -68,6 +67,32 @@ To return to using the default, public rekor instance:
 
    ./config.sh rekor-default
 
+### Local cluster rekor server
+
+After running the `bootstrap-cluster.sh` script, a rekor application is deployed to the local cluster. This rekor instance has an internal hostname and ingress configured ( `rekor.enterprise-contract-service.svc` ) which can be used in conjunction with chains. This hostname will not be accessible by default from your local development machine.
+
+Execute the following commands to update the chains configmap to allow chains to utilize this internal hostname instead of the public Sigstore instance:
+
+    # turn off gitops syncing (otherwise, the configmap is overwritten to ensure proper sync state)
+    $ ./hack/chains/gitops-sync off
+
+    # patch the configmap 
+    $ kubectl patch configmap chains-config -n tekton-chains \
+    -p='{"data":{"transparency.url": "http://rekor.enterprise-contract-service.svc:3000"}}'
```

Comment:
This is fine, but we could introduce a new config.sh command for this to make it easier, e.g.:

    ./hack/chains/config.sh rekor-internal

Reply:
Not a blocker, just a suggestion to consider.